secure_software_development_fundamentals.md

```diff
@@ -5116,6 +5127,8 @@ Open Web Application Security Project (OWASP), *Vulnerability Disclosure Cheat S
 
 Qualys, *SSL Server Test* ([https://www.ssllabs.com/ssltest/](https://www.ssllabs.com/ssltest/))
 
+Petro, Dan, and Allan Cecil, 2021, You're Doing IoT RNG, DEF CON 29 ([https://labs.bishopfox.com/tech-blog/youre-doing-iot-rng](https://labs.bishopfox.com/tech-blog/youre-doing-iot-rng)) with presentation at [https://www.youtube.com/watch?v=Zuqw0-jZh9Y](https://www.youtube.com/watch?v=Zuqw0-jZh9Y)
```

Comment:
Petro, Dan; Cecil, Allan;   ?

Reply:
As stated is correct. The ", and" means the next item on the list is the last item, and last item is the person's name (Allan Cecil).

Reply:
Oh wait, I see. The second comma shouldn't be there. I'll fix it.